src/plugins/cosmos/components/modals/Staking/views/ClaimBroadcast.tsx

```diff
@@ -0,0 +1,129 @@
+import { InfoOutlineIcon } from '@chakra-ui/icons'
+import { Flex } from '@chakra-ui/layout'
+import { Button, Link, ModalCloseButton, Text as CText, Tooltip } from '@chakra-ui/react'
+import { CAIP19 } from '@shapeshiftoss/caip'
+import { Asset } from '@shapeshiftoss/types'
+import { chainAdapters } from '@shapeshiftoss/types'
+import { useTranslate } from 'react-polyglot'
+import { Amount } from 'components/Amount/Amount'
+import { MiddleEllipsis } from 'components/MiddleEllipsis/MiddleEllipsis'
+import { SlideTransition } from 'components/SlideTransition'
+import { Text } from 'components/Text'
+import { useModal } from 'context/ModalProvider/ModalProvider'
+import { BigNumber } from 'lib/bignumber/bignumber'
+
+type ClaimBroadcastProps = {
+  assetId: CAIP19
+  cryptoStakeAmount: BigNumber
+  fiatAmountAvailable: string
+  isLoading: boolean
+}
+
+export enum Field {
+  FeeType = 'feeType'
+}
+
+export type ClaimBroadcastParams = {
+  [Field.FeeType]: chainAdapters.FeeDataKey
+}
+
+// TODO: Wire up the whole component with staked data
+export const ClaimBroadcast = ({
+  assetId,
+  cryptoStakeAmount,
+  fiatAmountAvailable,
+  isLoading
+}: ClaimBroadcastProps) => {
+  const { cosmosStaking } = useModal()
+
+  const translate = useTranslate()
+
+  const handleClose = () => {
+    cosmosStaking.close()
+  }
```

Comment:
```suggestion
  const handleClose = cosmosStaking.close
  }
```

purely stylistic, but i see this pattern all the time, creates another closure and stack frame and two extra lines

Reply:
agreed ! done